### DISCUSSION TITLE
What version of Magick.NET merges the ImageMagick/issue/592 fix?

### DISCUSSION THREAD

**Discussion by humor4fun** [dlemstra/Magick.NET#607] [General] [answered]
Can you confirm which version of Magic.NET merged in the fix for this memory leak issue: https://github.com/ImageMagick/ImageMagick/issues/592
Numerous release notes for Magic.NET only reference "Fixed numerous memory leaks" but don't refer to specific ImageMagick issue numbers.

**Comment by dlemstra** [comment, marked as answer]
I don't know which exact version solved that but most likely one the first one that was released after that date.

**Reply by humor4fun** [reply]
Alright. Thanks!
